src/fidesops/schemas/saas/saas_config.py

```diff
@@ -169,6 +169,7 @@ class ConnectorParam(BaseModel):
     """Used to define the required parameters for the connector (user-provided and constants)"""
 
     name: str
+    description: Optional[str]
```

Comment:
Adding this to include descriptions for fields that need more of an explanation than what can just be assumed from the name alone. Currently there is no additional logic in Fidesops that makes use of this information but this could be helpful when for when we have a UI to create/manage SaaS configs.